packages/sn-filter-pane/src/qae/object-definition.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @namespace properties
+ * @entry
+ */
+const objectDefinition = () => {
+  /**
+   * @lends properties
+   */
+  const definition = {
+    /**
+     * Current version of this generic object definition
+     * @type {string}
+     * @default
+     */
+    version: process.env.PACKAGE_VERSION,
+    /**
+     * Show visualization details toggle. (Only for sense-client)
+     * @type {boolean=}
+     * @default
+     * @private
+     */
+    showDetails: true,
+    /**
+     * Show title for the visualization.
+     * @type {boolean=}
+     * @default
+     */
+    showTitles: false,
+    /**
+     * Visualization title.
+     * @type {(string|StringExpression)=}
+     * @default
+     */
+    title: '',
+    /**
+     * Visualization subtitle.
+     * @type {(string|StringExpression)=}
+     * @default
+     */
+    subtitle: '',
+    /**
+     * Visualization footnote.
+     * @type {(string|StringExpression)=}
+     * @default
+     */
+    footnote: '',
+    /**
+     * Child object definition
+     * @type {ChildListDef}
+     * @default {}
+     * @description See {@link https://qlik.dev/apis/javascript/nebula-listbox-properties | Listbox properties}
```

Comment:
the qChildListDef do not contains the nebula-listbox-properties.
they are in the actual child object (included in the spec)